### ISSUE TITLE
Review licensing on whole L23 data microsite

### ISSUE THREAD

**Issue by ODILMichelle** [open-innovations/leeds-2023#89]
It is currently copyright Leeds2023 - do we need to make that CC-BY or similar in the spirit of being open?

**Comment by gilesdring** [comment]
Yes, probably. This occurred to me as I put the site footer together. Their main site is (c), so this has been transferred across. I'd prefer it to be CC-BY, and there's some justification for the legacy to make it an open license. This ought to apply to the data on the site. The code is already licensed MIT, in the repo.

**Comment by ODILMichelle** [comment]
Interesting, hadn't meant to add this to sprint 2.
________________________________

**Comment by gilesdring** [comment]
Should be included in wrap up

**Comment by taz-chiles** [comment]
@gilesdring do we need to review this?
